### PR TITLE
[DOCS] Make README more welcoming

### PR DESCRIPTION
**Pull request by bowenliang123**
### _Why are the changes needed?_
 

- Make Readme more welcoming to readers, in 
  - Elegancy: clear, center aligned Kyuubi logo in front
  - Maturity: always good to see numbers of contributors, lines and PRs
  - Guidance: quick starting point to project homepage and documentation for next steps
  - Sociability: showing star number of the project

Preview link: https://github.com/bowenliang123/incubator-kyuubi/blob/readme-welcome/README.md

Before:
image

After:
image

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [x] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/master/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by Simran180** [comment]
can I work on this issue?

**Comment by bowenliang123** [comment]
> can I work on this issue?

Hi, this is a pull request, including the changes for the improvement. This PR is intended to bring changes in styles and not in contents. Any suggestion is welcomed to comment on the lines of changed files.

File: README.md [hunk 1]
```diff
@@ -15,27 +15,43 @@
...
+  <a href="https://kyuubi.apache.org/releases.html">
+    <img src="https://img.shields.io/github/v/release/apache/kyuubi" />
+  </a>
+  <a href="https://github.com/apache/kyuubi/pulls">
+    <img src="https://img.shields.io/github/issues-pr-closed/apache/kyuubi" />
+  </a>
+  <a href="https://github.com/apache/kyuubi/pulse">
+    <img src="https://img.shields.io/tokei/lines/github/apache/kyuubi" />
+  </a>
+  <a href="https://github.com/apache/kyuubi/graphs/contributors">
```
Comment (yaooqinn) [inline README.md:32]: The stats of PRs, contributors, and stars seem to have been provided by GitHub on the page

**Comment by yaooqinn** [comment]
@Simran180 this issue has a PR already. You can check the issues tagged with 'help wanted'

File: README.md (hunk 1 above)
Comment (bowenliang123) [reply README.md:32]: Yes, it more about highlights and interations.
The whole line with all the badges comes to "Hi, I'm Kyuubi. I are ready as in latest released version in 1.6.1-incubating and later, which conducted in 2.5k PRs and resulted in 22K total links, and all the commits are done by the community of 116 contributures. Welcome to mark this project starred and see me later."

Comment (bowenliang123) [reply README.md:32]: Or any better suggestion for the item and order of badges list?

Comment (yaooqinn) [reply README.md:32]: Sound fine to me.

BTW, can we move those badges for project maintenance to https://github.com/apache/kyuubi/blob/master/CONTRIBUTING.md, PMCs somehow rely on them to evaluate the community healthiness

Comment (bowenliang123) [reply README.md:32]: Agree, or let's do it in the follow-up PR to bring missing badges to `CONTRIBUTING.md`.

**Review by yaooqinn — APPROVED** [review]

**Review by cxzl25 — APPROVED** [review]

**Comment by bowenliang123** [comment]
Thanks, merged to master.

**Pull request closed without merging**